ui/pages/settings/networks-tab/networks-form/networks-form.js

```diff
@@ -551,6 +551,7 @@ const NetworksForm = ({
           iconFillColor="var(--color-warning-default)"
           useIcon
           withRightButton
+          className="networks-tab__add-network-form__alert"
         />
       ) : null}
       <div
```

Comment:
Adding class to remove unnecessary margin

## Before
<img width="648" alt="Screen Shot 2022-06-14 at 4 42 29 PM" src="https://user-images.githubusercontent.com/8112138/173707163-f017ad8f-e075-4932-897f-8319865f8861.png">

## After

<img width="722" alt="Screen Shot 2022-06-14 at 4 42 53 PM" src="https://user-images.githubusercontent.com/8112138/173707164-47d44c78-9dd8-4d85-996d-e77a54a17c69.png">